File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/WorkspaceEditParams.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class WorkspaceEditParams(
  val operations: List<WorkspaceEditOperation>,
  val metadata: WorkspaceEditMetadata? = null,
)

File: agent/bindings/csharp/WorkspaceEditParams.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class WorkspaceEditParams
  {
    [JsonProperty(PropertyName = "operations")]
    public WorkspaceEditOperation[] Operations { get; set; }
    [JsonProperty(PropertyName = "metadata")]
    public WorkspaceEditMetadata Metadata { get; set; }
  }
}
